ui/css/oae.signup.css

```diff
@@ -28,38 +36,142 @@ main > h3 {
     padding-bottom: 10px;
 }
 
+#signup-institution-container,
+#signup-options-container {
+    margin: 0 20px;
+    padding: 40px;
+}
+
+#signup-institution,
+#signup-options {
+    margin: auto;
+    max-width: 992px;
+}
+
+
+/**************************
+ ** INSTITUTIONAL SEARCH **
+ **************************/
+
+/** Layout **/
+
+#signup-institution-container {
+    border-bottom: solid 1px #DADADA;
+    display: none;
+}
+
+#signup-institution {
+    display: -webkit-flex;
+    display:  -ms-flexbox;
+    display:         flex;
+}
+
+#signup-institution > div {
+    /* IE10 defaults to 1 0 auto, so set this */
+    -ms-flex: 0 1 auto;
+}
+
+#signup-institution-instruction {
+    margin-right: 50px;
+    min-width: 250px;
+}
+
+#signup-institution-search {
+    /* This `min-width` is needed to ensure the item doesn't expand beyond the
+       bounds of its parent when the tenant name is extremely long */
+    min-width: 0;
+    width: 100%;
+}
+
+#signup-institution-search > label {
+    width: 100%;
+}
+
+/** Suggestion items **/
+
+#signup-institution-search ul.as-selections.signup-institution-search-loading {
+    background-color: #EFEFEF;
+}
+
+#signup-institution-search ul.as-selections.signup-institution-search-loading > li {
+    line-height: 35px;
+    width: 100%;
+}
+
+#signup-institution-search ul.as-selections.signup-institution-search-loading > li > i {
+    display: block;
+    float: left;
+    line-height: 35px;
+    margin: -35px 10px 0 10px;
+}
+
+#signup-institution-search ul.as-selections.signup-institution-search-loading > li > div {
+    padding-left: 40px;
+}
+
+#signup-institution-search .as-selection-item {
+    display: none;
+}
+
+#signup-institution-search .as-results {
+    position: relative;
+}
+
+#signup-institution-search ul.as-list {
+    width: 100% !important;
+}
+
+#signup-institution-search ul.as-list li.as-result-item {
+    /* Replace padding with table border spacing */
+    padding: 0;
+}
+
+#signup-institution-search .signup-tenant-item {
+    border-spacing: 10px;
+    display: table;
+    width: 100%;
+}
+
+#signup-institution-search .signup-tenant-item > .signup-tenant-item-inner {
+    display: table-row;
+}
+
+#signup-institution-search .signup-tenant-item > .signup-tenant-item-inner > div {
+    display: table-cell;
+    vertical-align: middle;
+}
+
+#signup-institution-search .signup-tenant-flag {
+    text-align: center;
+    width: 60px;
```

Comment:
We can reduce this to `40px` so tenants without a flag don't have too much whitespace